.github/workflows/aomp-shell.yml

```
@@ -0,0 +1,16 @@
+name: AOMP Lint
+
+on:
+  push:
+  pull_request:
+  
+jobs:
+  aomp-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Run shellcheck
+        run: | 
+          find . -name "*.sh" | xargs shellcheck -S info {} \;
```

The syntax is wrong, which must be
```suggestion
          find . -name "*.sh" -exec shellcheck -S info {} +
```
or
```suggestion
          find . -name "*.sh" | xargs shellcheck -S info
```